app/models/drug.rb

```diff
@@ -0,0 +1,38 @@
+class Drug < ActiveYaml::Base
```

Comment:
Shall we call this model `Medication`, given we have a chance to rename and this is going to be the long term model? This seems to be the FHIR equivalent of what we're modelling https://www.hl7.org/fhir/medication.html.

Aside: `PrescriptionDrug` = https://www.hl7.org/fhir/medicationrequest.html, although this rename might be too much 😅 .

Reply:
Hmm, that's a good call, will take a look at FHIR models

Reply:
Renamed 🙌